ledger/sandbox-classic/src/test/resources/logback-test.xml

```diff
@@ -13,28 +13,16 @@
         <appender-ref ref="SqlLedgerSpecAppender" />
     </logger>
 
-    <logger name="com.daml.platform.store.dao.HikariConnection" level="INFO">
```

Comment:
Why are we removing this and the following definitions for `SqlLedgerSpecAppender`?

Reply:
Because the test case no longer makes sense for `SqlLedgerSpec`. We do need to verify this but I have no idea how we should go about it, as we don't log anything in `PostgresStorageBackend`, and even if we did, it's only triggered on subscriptions, which is not what we're doing here.

I would hope that as `PostgresStorageBackend` is new code, it's well-tested.

Reply:
I see it now, thanks, makes sense